tipg/settings.py

```diff
@@ -141,6 +141,8 @@ class PostgresSettings(BaseSettings):
 
     model_config = {"env_file": ".env", "extra": "ignore"}
 
+    tipg_schema: str = Field("pg_temp", pattern=r"[a-zA-Z][a-zA-Z0-9_-]*")
```

Comment:
- can you move this up, before the model_config 
- what do you think about `TIPG_SCHEMA_NAME` ?